smac/utils/sklearn_wrapper.py

```diff
@@ -0,0 +1,329 @@
+"""
+author = "Jan N. van Rijn"
+license = "3-clause BSD"
+"""
+
+import numpy as np
+
+from collections import defaultdict
+from functools import partial
+from typing import Union, Callable, Dict, List
+
+from ConfigSpace import Configuration
+from ConfigSpace.hyperparameters import CategoricalHyperparameter, UniformIntegerHyperparameter, UniformFloatHyperparameter
+
+from sklearn.base import is_classifier, clone, BaseEstimator
+from sklearn.model_selection._split import check_cv, BaseCrossValidator
+from sklearn.model_selection._search import BaseSearchCV
+from sklearn.model_selection._validation import _fit_and_score
+from sklearn.utils.fixes import rankdata
+from sklearn.utils.fixes import MaskedArray
+from sklearn.utils.validation import indexable
+from sklearn.metrics.scorer import check_scoring
+
+from smac.configspace import ConfigurationSpace
+from smac.facade.smac_facade import SMAC
+from smac.scenario.scenario import Scenario
+
+
+class ModelBasedOptimization(BaseSearchCV):
+    """
+    Scikit-Learn wrapper for SMAC
+    """
+    def __init__(self, estimator: BaseEstimator, param_distributions: Dict[str, List], n_iter: int = 10,
```

Comment:
Probably. If there is a SMAC default I would be happy to comply to that one? For now 200 it is.